entries/a/availity.com.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Availity Essentials": {
+    "domain": "availity.com",
+    "tfa": [
+      "sms",
+      "call",
+      "totp"
+    ],
```

Comment:
Do they also support U2F?

> 2-step enrollment requires you to register your preferred authentication method through your choice
of text, voice, mobile authenticator app (i.e., Google Authenticator), or a hard token, also known as a
security token or authorization token.